packages/widgets/src/blocks/legacy-widget/edit/index.js

```diff
@@ -57,30 +57,32 @@ function Empty( { attributes: { id, idBase }, setAttributes } ) {
 			icon={ <BlockIcon icon={ brushIcon } /> }
 			label={ __( 'Legacy Widget' ) }
 		>
-			<WidgetTypeSelector
-				selectedId={ id ?? idBase }
-				onSelect={ ( { selectedId, isMulti } ) => {
-					if ( ! selectedId ) {
-						setAttributes( {
-							id: null,
-							idBase: null,
-							instance: null,
-						} );
-					} else if ( isMulti ) {
-						setAttributes( {
-							id: null,
-							idBase: selectedId,
-							instance: {},
-						} );
-					} else {
-						setAttributes( {
-							id: selectedId,
-							idBase: null,
-							instance: null,
-						} );
-					}
-				} }
-			/>
+			<div className="wp-block-legacy-widget__selector-container">
```

Comment:
Btw there is also a `FlexBlock` component you can use for this purpose. (Sorry for all these undiscoverable components 😅)

Reply:
Don't be sorry! It's super valuable for me to know, and demonstrates why this project is such a great learning experience! 😄